multicluster/apis/multicluster/v1alpha1/clusterset_types.go

```diff
@@ -47,10 +45,73 @@ type ClusterSetSpec struct {
 	Namespace string `json:"namespace,omitempty"`
 }
 
-// ClusterSetStatus defines the observed state of ClusterSet
+type ClusterSetConditionType string
+
+const (
+	// ClusterSetReady indicates whether ClusterSet is ready.
+	ClusterSetReady ClusterSetConditionType = "ClusterSetReady"
```

Comment:
I feel like this is typically just "Ready" in K8s APIs. Am I wrong? Maybe we can avoid repeating the resource name in the status name: Ready, IsLeader, ImportSucceeded, etc.